waltid-libraries/waltid-openid4vc/src/jvmTest/kotlin/id/walt/oid4vc/CI_JVM_Test.kt

```diff
@@ -612,6 +614,1065 @@ class CI_JVM_Test {
         assertTrue(actual = JwtSignaturePolicy().verify(credential, null, mapOf()).isSuccess)
     }
 
+    // Test case for available authentication methods are: NONE, ID_TOKEN, VP_TOKEN, PRE_AUTHORIZED PWD(Handled by third party authorization server)
+    @Test
+    fun testCredentialIssuanceIsolatedFunctionsAuthCodeFlow() = runTest {
```

Comment:
test is about 500 lines long, maybe split into multiple tests?

Reply:
From what I understand, this is an amalgamated test case that runs through id and vp token flows in the context of the same function. Perhaps it would be reasonable to split it up when the corresponding changes are pushed to the wallet api as well?